designs/jane/src/sleeve.mjs

```diff
@@ -3,7 +3,7 @@ export const sleeve = {
   measurements: ['biceps', 'shoulderToElbow'],
   options: {
     bicepsEase: { pct: 18, min: 18, max: 51, menu: 'fit' },
-    sleeveLength: { pct: 80, min: 60, max: 100, menu: 'style' },
+    sleeveLengthBonus: { pct: 80, min: 60, max: 100, menu: 'style' },
```

Comment:
It is not a `bonus` when the minimum is 60%. Which is why I changed the option because the name is misleading.

Not sure if you saw [my comment](https://github.com/freesewing/freesewing/pull/6738#issuecomment-2106192386), but this is not an accident.